EntAppProject/JBCreateAccount.html

```diff
@@ -1,53 +1,33 @@
-<!DOCTYPE html>
-
-<html>
-
-<head>
-
-<title>JobBook - Create an Account</title>
-
-<link rel="stylesheet" href="JobBookCSS.css">
- 
-</head>
-
-<body>
- 
-<h1><span>Job</span>Book</h1>
-
-<h2>Create an Account</h2>
-
-<br>
-
-<fieldset>
-
-<form action="file:///C:/Users/Administrator/eclipse-workspace/jobBookFINAL/src/main/resources/templates/JBHomescreen.html" method="post">
-
-<div class="container">
-        <label for="useremail"><b>Email:</b></label>
-        <input type="email" placeholder="Email:" name="useremail" required>
-        <br>
-        <label for="username"><b>Username:</b></label>
-        <input type="text" placeholder="Username:" name="username" required>
-        <br>
-        <br>
-        <label for="userpassword"><b>Password:</b></label>
-        <input type="password" placeholder="Password" name="userpassword" required>
-        <br>
-        <label for="userpasswordconfirm"><b>Confirm Password:</b></label>
-        <input type="password" placeholder="Password" name="userpasswordconfirm" required>
-        <br>
-        <button type="submit">Create</button>
-        <label>
-        <br>
-        <input type="checkbox" checked="checked" name="remember"> Remember me?
-        </label>
-        </div>
-        <br>
-        <br>
-        </form>
- 
-</fieldset>
- 
-</body>
-
+<!DOCTYPE html>
+
+<html>
+
+        <head>
+                <title>JobBook - Create an Account</title>
+                <link rel="stylesheet" href="JobBookCSS.css">
+        </head>
+
+        <body>
+                <h1><span>Job</span>Book</h1>
+                <h2>Create an Account</h2>
+                <br>
+
+                <form action="file:///C:/Users/Administrator/eclipse-workspace/jobBookFINAL/src/main/resources/templates/JBHomescreen.html" method="post">
+                        <fieldset>
+                                <div class="container">
+                                        <label for="useremail"><b>Email:</b></label>
```

Comment:
`<label for="THIS_IS_THE_ID_OF_AN_HTML_MARKUP">` takes the `id` of another html markup to create the association between a label and an input box. Generally, I only use this feature when i am working with radio buttons so that the UI reflects which radio I have selected, but for the purposes of this code review, I simply added the `id` property to the respective HTML tags